www/creating-zooming-images.html

```diff
@@ -191,6 +196,44 @@ <h2>Creating Zooming Images</h2>
         <td>Free desktop app for locally viewing DZI files for Windows. Linux soon.</td>
         <td><a href="/examples/tilesource-dzi/">DZI</a></td>
     </tr>
+    
+    <tr>
+        <td colspan="3"><h4>Single-file Tile formats - see complete <a href="https://github.com/openseadragon/openseadragon/issues/944">discussion thread</a>. Useful for faster/easier file operations and maintenance on S3/file servers FS.</h4></td>
+    </tr>
+    <tr>
+        <td>
+            szi_explorer <a href="https://github.com/infovore/szi_explorer">repo</a> and <a href="https://szi-explorer.netlify.app">demo</a>
+        </td>
+        <td>SZI is losslessly zipped DeepZoom.dzi container, and szi_explorer is the current OSD implementation by infovore and PolCPP. SZI can be produced with <a href="https://www.libvips.org/API/current/making-image-pyramids.html">libvips</a></td>
+        <td><a href="https://github.com/smartinmedia/SZI-Format/blob/master/SZI%20format%20description%20-%202018-11-24.pdf">SZI</a></td>
+    </tr>
+    <tr>
+        <td>
+            OpenSeaDragon <a href="https://github.com/pearcetm/GeoTIFFTileSource">GeoTIFFTileSource</a> and <a href="https://pearcetm.github.io/GeoTIFFTileSource/demo/demo.html">demo</a>
+        </td>
+        <td>Cloud Optimized Geotiff is a geospatial standard specified by OGC for massive rasters, which also works for non-geo images</td>
+        <td><a href="https://www.cogeo.org/">COG</a></td>
+    </tr>
+    <tr>
+        <td>WebGIS alternatives to OSD for viewing COGs</td>
```

Comment:
I don't think this section is really appropriate for this page on creating content for OSD. This information on other viewers probably belongs somewhere else in the ecosystem.